test_tictactoe.py

```diff
@@ -25,7 +25,14 @@ def test_computers_move(self):
         self.fail("not implemented")
 
     def test_check_winner(self):
-        self.fail("not implemented")
-
+	def test_human_wins(self):
+		self.assertEqual(check_winner('XXXEOEEOO'), True)	
```

Comment:
The return values are wrong. They should be the 4 strings in the `check_winner()` docstring.